src/Services/QueuedJobService.php

```diff
@@ -917,6 +966,26 @@ public function runJob($jobId)
         return !$broken;
     }
 
+    /**
+     * @param QueuedJobDescriptor $jobDescriptor
+     * @param QueuedJob $job
+     * @param $e
+     * @throws ValidationException
```

Comment:
We don't usually document exceptions unless the function explicitly throws them itself, I guess this comes from the write() call, but maybe omit it from here